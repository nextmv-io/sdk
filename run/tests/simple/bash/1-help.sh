go run main.go --help
