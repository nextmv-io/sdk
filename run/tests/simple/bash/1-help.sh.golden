Nextmv Hybrid Optimization Platform
Usage:
  -duration duration
    	Sleep duration. (env DURATION) (default 1s)
  -runner.input.path string
    	The input file path (env RUNNER_INPUT_PATH)
  -runner.output.path string
    	The output file path (env RUNNER_OUTPUT_PATH)
  -runner.output.solutions string
    	{all, last} (env RUNNER_OUTPUT_SOLUTIONS) (default "last")
  -runner.profile.cpu string
    	The CPU profile file path (env RUNNER_PROFILE_CPU)
  -runner.profile.memory string
    	The memory profile file path (env RUNNER_PROFILE_MEMORY)
